### PR TITLE
Add cj-switch plugin to krew index

### PR DESCRIPTION
**Pull request by pymag09**
I've read guide and tested it locally.

Default behavior of cj-switch is inverting the 'spec.suspend' value of a cron job ('True' turns to 'False' and vice versa). It is also possible to set 'spec.suspend' to True or False explicitly. If you need to preserve previous state and revert it later, please use -f key (for instance when some jobs are suspended, some are not).

### CODE REVIEW

**Comment by pymag09** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> memo **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
>     * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
>     * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> 
>     * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> 
>     * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

I signed the CLA

File: plugins/cj-switch.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  description: |
+    The list of the features:
+    * Invert the spec.suspend flag. 'True' turns to 'False' and vice versa
+    * Save current state to and restore state from file
+    * Overwrite spec.suspend flag
+    Possible filters:
+      * All namespaces
+      * Filter by label
+      * Filter by namespace
+    Example:
```
Comment (ahmetb) [inline plugins/cj-switch.yaml:15]: Please move usages to -help of your plugin. These will be outdated.

File: plugins/cj-switch.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    files:
+    - from: /*
+      to: .
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/pymag09/kubectl_cj-switch/releases/download/v1.0.0/cj-switch-v1.0.0.tar.gz
+    sha256: 72fa9f5c5304c458e97e7751aa7963ea4177f2d74dd89b09c9bcb2664f6bfb6d
+  shortDescription: Used to enable/disable cronjobs
```
Comment (ahmetb) [inline plugins/cj-switch.yaml:44]: Remove "Used to"

Better to capitalize as "CronJobs" akin to Kubernetes api.

File: plugins/cj-switch.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cj-switch
```
Comment (ahmetb) [inline plugins/cj-switch.yaml:4]: Comment-only: Have you considered a name like cj-toggle or cronjob-toggle or toggle-cronjob?

This is not a requirement but I think your chances of discovery would be better.

**Review by ahmetb — CHANGES_REQUESTED** [review]

File: plugins/cj-switch.yaml (hunk 3 above)
Comment (pymag09) [reply plugins/cj-switch.yaml:4]: Thank you for the hint. That's much better than switch.

File: plugins/cj-switch.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      #
+      # Restore state from the file
+      #
+      kubectl cj-switch -f ./<FILENAME> -r
+  homepage: https://github.com/pymag09/kubectl_cj-switch
+  platforms:
+  - bin: "./cj-switch.sh"
+    files:
+    - from: /*
+      to: .
```
Comment (corneliusweig) [inline plugins/cj-switch.yaml:37]: This is the default. No need to spell it out.

File: plugins/cj-switch.yaml (hunk 1 above)
Comment (corneliusweig) [reply plugins/cj-switch.yaml:15]: +1

File: plugins/cj-switch.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+spec:
+  description: |
+    The list of the features:
+    * Invert the spec.suspend flag. 'True' turns to 'False' and vice versa
+    * Save current state to and restore state from file
+    * Overwrite spec.suspend flag
+    Possible filters:
+      * All namespaces
+      * Filter by label
+      * Filter by namespace
```
Comment (corneliusweig) [inline plugins/cj-switch.yaml:14]: Just a suggestion:
```suggestion
    The plugin manipulates the suspend state of CronJobs. You can:
    * Toggle the spec.suspend flag.
    * Save current state to and restore state from a file
    * Overwrite the spec.suspend flag
    You can select CronJobs by:
      * By label
      * By namespace
      * Across all namespaces
```

File: plugins/cj-switch.yaml (hunk 4 above)
Comment (pymag09) [reply plugins/cj-switch.yaml:37]: Unfortunately without this i see the following error:
> Selector:nil Files:[{From: To:}] Bin:./cj-toggle.sh}) is badly constructed: `files` is invalid: `from` field has to be

Comment (corneliusweig) [reply plugins/cj-switch.yaml:37]: 🤔 did you drop only one of `from` and `to`? This error is peculiar.

Comment (pymag09) [reply plugins/cj-switch.yaml]: I dropped both but I left `files:` Probably this caused the error.

**Comment by ahmetb** [comment]
CLA is still not resolved.

**Comment by pymag09** [comment]
> CLA is still not resolved.

Does CLA somehow depends on License I choose? I mean I choose Apache. I've run though CLA and as far as I understand ti si about copyrights.What if I change license to make it completly free, do I need to complete CLA?

**Comment by ahmetb** [comment]
Not related to the license. There should be links above explaining it and how to sign it.

**Pull request closed without merging**